spiceaidocs/docs/data-connectors/mysql.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'MySQL Data Connector'
+sidebar_label: 'MySQL Data Connector'
+description: 'MySQL Data Connector Documentation'
+---
+
+## Federated SQL query
+
+To connect to any MySQL database as connector for federated SQL query, specify `mysql` as the selector in the `from` value for the dataset.
+
+```yaml
+datasets:
+  - from: mysql:path.to.my_dataset
+    name: my_dataset
+```
+
+## Configuration
+
+The connection to MySQ: can be configured by providing the following `params`:
+
+- `mysql_connection_string`: The connection string to use to connect to the MySQL server. This can be used instead of providing individual connection parameters.
+- `mysql_connection_string_key`: The secret key containing the connection string to use to connect to the MySQL server. This can be used instead of providing indivi                ';[[[[';/\[dual connection parameters.
```

Comment:
```suggestion
- `mysql_connection_string_key`: The secret key containing the connection string to use to connect to the MySQL server. This can be used instead of providing individual connection parameters.
```